st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -105,9 +105,7 @@ def get_one(
 
         if is_admin and user_query_param_filter:
             # Retrieve values scoped to the provided user
-            user_scope_prefix = get_key_reference(
-                name=name, scope=USER_SCOPE, user=user
-            )
+            user_scope_prefix = get_key_reference(name=name, scope=scope, user=user)
```

Comment:
@m4dcoder - Now the scope is dynamically passed, so when the scope is not passed then it will return FULL_SYSTEM_SCOPE. Check for scope==USER_SCOPE is already handled in the method.
Screenshot for your reference. Please let us know your suggestions.
![keyvalue](https://user-images.githubusercontent.com/72917414/133470535-4d6ac8eb-473a-4920-b22b-8f420cbc8737.jpg)

Reply:
Where is scope==USER_SCOPE handled?  Can you give me the link on your branch?  Please don't screenshot. I need to trace the code.